Source/Core/Core/IOS/FS/FileSystemProxy.cpp

```diff
@@ -46,6 +51,9 @@ FS::FS(Kernel& ios, const std::string& device_name) : Device(ios, device_name)
 void FS::DoState(PointerWrap& p)
 {
   p.Do(m_fd_map);
+  p.Do(m_cache_fd);
+  p.Do(m_cache_chain_index);
+  p.Do(m_dirty_cache);
```

Comment:
State change, needs a bump in the version

Reply:
The version is already bumped.

Reply:
I...don't see a change of the version in the remaining commits? Or is GitHub being silly?

Reply:
Bumped it in ed822ce (from 95 (PR 6421) to 96)